src/runtime/crt/host/main.cc

```diff
@@ -110,7 +110,7 @@ tvm_crt_error_t TVMPlatformGenerateRandom(uint8_t* buffer, size_t num_bytes) {
 }
 }
 
-uint8_t memory[512 * 1024];
+uint8_t memory[1024 * 1024];
```

Comment:
why 1024 is good enough? Is it just a fix to make some use case not crash or it is some reasons that solves issue for any use case?

Reply:
This is specific for crt runtime. I'm running few models and it crashes in those cases.